ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.component.ts

```diff
@@ -195,8 +194,7 @@ export class ComputeCapacityComponent implements OnInit {
                     // user selects a host or cluster that has less than 2048MB of ram
                     // available for VM endpoint. as a solution, we set the smaller
                     // value between 2048 and memory['maxUsage']
-                    this.form.get('endpointMemory').setValue(
-                        Math.min(memory['maxUsage'], 2048) + '')
+                    this.form.get('endpointMemory').setValue(Math.min(memory['maxUsage'], 2048) + '');
```

Comment:
Maybe this 2048 should also be a constant?

Reply:
Sure. Done.